rust/relay/server/src/main.rs

```diff
@@ -154,22 +163,49 @@ fn main() {
 async fn try_main(args: Args) -> Result<()> {
     let filter_reload_handle = setup_tracing(&args)?;
 
-    let mut ebpf = args
-        .ebpf_offloading
-        .as_deref()
-        .map(|interface| ebpf::Program::try_load(interface, args.ebpf_attach_mode))
-        .transpose()
-        .context("Failed to load eBPF TURN router")?;
-
-    if let Some(ebpf) = ebpf.as_mut() {
-        ebpf.set_config(
-            Config::default()
-                .with_udp_checksum(true)
-                .with_lowest_allocation_port(args.lowest_port)
-                .with_highest_allocation_port(args.highest_port),
+    let ebpf = if let Some(interface) = args.ebpf_offloading.as_deref() {
+        match (args.ebpf_int4_addr, args.ebpf_int6_addr) {
```

Comment:
Wouldn't this be easier if we had two separate `if let Some` blocks?

Reply:
👍